### PR TITLE
edit description for is_file (php#3719)

### PR DESCRIPTION
**Pull request by vera-ju**

### CODE REVIEW

File: reference/filesystem/functions/is-file.xml [hunk 1]
```diff
@@ -13,7 +13,8 @@
    <methodparam><type>string</type><parameter>filename</parameter></methodparam>
   </methodsynopsis>
   <para>
-   Tells whether the given file is a regular file.
+   Tells whether the given file is a regular file. It will also return true if <parameter>filename</parameter> is a
+   symbolic link.
```
Comment (cmb69) [inline reference/filesystem/functions/is-file.xml:17]: I'm not sure about that; doesn't it check the symlink's target instead?

Comment (vera-ju) [reply reference/filesystem/functions/is-file.xml:17]: Oh yes, thank you. You are right. I will change the description again :-)

**Comment by cmb69** [comment]
This is now correct (I think), but I'm not sure how to handle this. Symlinks are usually resolved by all functions (there are some deliberate exceptions, though), but documenting that everywhere feels clumsy. Now, we have that already documented for `is_dir()`, but not for `file_exists()`. Maybe there should be some general info available somewhere?

@Girgias, thoughts?

**Review by Girgias — APPROVED** [review]
I think having a generic description of how file paths are resolved might be a good idea. But I will merge this PR as it is to get this moving forward.

**Pull request merged**